apis/common/v1/policies.go

```diff
@@ -16,14 +16,33 @@ limitations under the License.
 
 package v1
 
+// A ManagementPolicy determines how should Crossplane controllers manage an
+// external resource.
+// +kubebuilder:validation:Enum=FullControl;ObserveOnly;OrphanOnDelete
+type ManagementPolicy string
+
+const (
+	// ManagementFullControl means the external resource is fully controlled
+	// by Crossplane controllers, including its deletion.
+	ManagementFullControl ManagementPolicy = "FullControl"
+
+	// ManagementObserveOnly means the external resource will only be observed
```

Comment:
Should we give here info that the external resource has to already exist if this ManagementPolicy is chose, or is there a better place for that?

Reply:
Yeah it should be obvious.